css/style.css

```diff
@@ -1,6 +1,96 @@
 /* General page styles */
 
 /* We should use a mobile first approach having out breack point at 768px */
+/*Comodities */
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 20px;
+}
+
+.carousel-image{
+  width: 100%;
+  height: 100%;
+  max-height: 233px;
+  max-width: 367px;
+}
+.carrouselContainer {
```

Comment:
i see you use this class for a js action, maybe set `carrouselContainer` as an id for js use, and set this class to be `carrousel-container`

Reply:
better